fvtest/tril/tril/ilgen.cpp

```diff
@@ -287,8 +290,22 @@ TR::Node* Tril::TRLangBuilder::toTRNode(const ASTNode* const tree) {
                                                                                           targetAddress,
                                                                                           0);
 
-         TR::SymbolReference *methodSymRef = symRefTab()->findOrCreateStaticMethodSymbol(JITTED_METHOD_INDEX, -1, method);
-         node  = TR::Node::createWithSymRef(opcode.getOpCodeValue(), childCount, methodSymRef);
+        TR::SymbolReference *methodSymRef = symRefTab()->findOrCreateStaticMethodSymbol(JITTED_METHOD_INDEX, -1, method);
+
+        /* Calls can have a customized linkage */
+        const auto* linkageArg= tree->getArgByName("linkage");
+        if (linkageArg != NULL) { 
+           const auto* linkageString = linkageArg->getValue()->getString();
+           TR_LinkageConventions lc = convertStringToLinkage(linkageString); 
+           if (lc != TR_None) {
+              methodSymRef->getSymbol()->castToMethodSymbol()->setLinkage(lc);     
+              TraceIL("  customizing linakge of call to %s (lc=%d)\n", linkageString, lc);
+           } else { 
+              TraceIL("  failed to find customized linkage %s, aborting parsing\n", linkageString);
+              return NULL; 
+           }
+        }
+        node  = TR::Node::createWithSymRef(opcode.getOpCodeValue(), childCount, methodSymRef);
```

Comment:
If you're going to optionally override the linkage here I think I'd prefer the default linkage setting to be made explicitly here (even if it's redundant because it's already set somewhere else).

Reply:
Done.